core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -1324,7 +1323,7 @@ private void updateMediaSessionMetadata(final Playable p) {
                         try {
                             art = Glide.with(this)
                                     .asBitmap()
-                                    .load(ImageResourceUtils.getFallbackImageLocation(p))
+                                    .load(p.getImageLocation())
```

Comment:
The fallback here now does the same as the first try. I think there is something wrong.

Reply:
Oh no! I did several mistakes that change the code behaviour which is not intended by a refactoring. So I started from scratch again.